.github/workflows/sessions.yml

```diff
@@ -33,7 +33,9 @@ jobs:
       run: scripts/setup_bundler.sh
     - name: Build and test
       run: |
-        scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseSessions.podspec --platforms=${{ matrix.target }}
+        scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseSessions.podspec \
+            --platforms=${{ matrix.target }} \
+            --allow-warnings
```

Comment:
We should try hard not to disable warnings. Otherwise we'll risk a bunch of test escapes and issues generated

Reply:
removed `--allow-warnings `